less/index.less

```diff
@@ -53,7 +53,164 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html {
+	height: 100%;
+	font-size: 62.5%;
+	body{
+		font-size: 1.6rem;
+	}
+    
 }
 
+
+//Variables
+@background: #babbbc;
+@top-content-color: #aa2a4a;
+@letter-colors: white;
+
```

Comment:
Good work with you mixins.  Just a tip, you can also use variables inside of your mixins.  This can help with your css as well.